api/routes/api.php

```diff
@@ -18,3 +20,79 @@
     Route::post('/tickets', 'createTicket');
 });
 
+/**
+ * Note: These routes are for testing/example and should be
+ * removed before merging
+ */
+Route::prefix('notify')->group(function () {
+    $templates = config('notify.templates');
+
+    Route::get('email', function () use ($templates) {
+        try {
+            $response = Notify::sendEmail(
+                'domain@test.com', // Replace with your email
+                $templates['email'],
+                ['name' => 'Your Name']
+            );
+
+            return $response->json();
+        } catch (\Exception $e) {
+            echo $e->getMessage();
+        }
+    });
+
+    Route::get('email/attachment', function () use ($templates) {
+        try {
+            $file = Storage::get('example.txt'); // Add a file by this name to `api/storage/app`
+            $response = Notify::sendEmail(
+                'domain@test.com', // Replace with your email
+                $templates['email'],
+                ['name' => 'Your Name'],
+                null,
+                null,
+                [
+                    'file' => base64_encode($file),
+                    'filename' => "attachment.txt"
+                ]
+            );
```

Comment:
I don't like the double null's to add an attachment. Is anyone aware of a better way to do this or should be shuffle the order around to put `$attachment` ahead of `$reference` and `$replyTo`?

Reply:
I don't mind the double nulls, perhaps add a comment beside them?